test/pbkdf2_SUITE.erl

```diff
@@ -117,13 +116,17 @@ erlang_and_nif_are_equivalent_sha512(_Config) ->
 erlang_and_nif_are_equivalent_(Sha) ->
     Prop = ?FORALL({Pass, Salt, Count},
                    {binary(), binary(), range(2,20000)},
-                   fast_pbkdf2:pbkdf2(Sha, Pass, Salt, Count)
-                       =:= erl_pbkdf2:pbkdf2_oneblock(Sha, Pass, Salt, Count)
-                  ),
-    ?assert(proper:quickcheck(Prop, [verbose, long_result,
-                                     {numtests, 100},
-                                     {start_size, 2},
-                                     {max_size, 64}])).
+                   begin
+                       #{size := KeyLen} = crypto:hash_info(Sha),
+                       This = fast_pbkdf2:pbkdf2(Sha, Pass, Salt, Count),
+                       PureErl = erl_pbkdf2:pbkdf2_oneblock(Sha, Pass, Salt, Count),
```

Comment:
what is the sense of having erl_pbkdf2 module, don't yu trust standard crypto library? :)

Reply:
Actually, nope :D It just calls OpenSSL pbkdf2 function on a regular scheduler without yielding, so it's buggy 😄